airflow/config_templates/default_airflow.cfg

```diff
@@ -67,6 +67,9 @@ colored_formatter_class = airflow.utils.log.colored_log.CustomTTYColoredFormatte
 log_format = [%%(asctime)s] {{%%(filename)s:%%(lineno)d}} %%(levelname)s - %%(message)s
 simple_log_format = %%(asctime)s %%(levelname)s - %%(message)s
 
+# Colored logs in the WebUI (ANSI Colour)
+colored_ui_logs = False
```

Comment:
New config options should be added to airflow/config_templates/config.yml which will then auto-generate this file and our docs.

Reply:
You still haven't done this change.

Reply:
@ashb, I dont have config.yml file in this path, should I create new file?

![image](https://user-images.githubusercontent.com/6237072/75980429-1d4afa80-5eeb-11ea-979d-f1f600fca886.png)